infrastructure/README.md

```diff
@@ -0,0 +1,54 @@
+# Infrastructure
+
+It is possible to deploy the Optic application to the Google Cloud Platform (GCP) and automate its releases.
+
+The [Terraform](https://www.terraform.io/) scripts into the `infrastructure/` folder let you to create
+the infrastructure needed.
+
+It creates the following resources:
+
+- a Cloud Run application.
+- one Service Account to be used by the Cloud Run application to serve the Optic application.
+- one Service Account to be used by GitHub to automate the deployment.
+
+## Prerequisites
+
+Before proceeding with the installation, you need to have the following requirements:
+
+1. Create a Firebase project as [described here](../README.md#prerequisites).
+1. Create a Service Account:
```

Comment:
Markdown numbering looks a bit off

Reply:
Ignore me - didn't check the render just assumed it would display correctly in the md file!